File: app/src/main/java/com/utynote/components/search/model/geojson/entities/Lang.java

package com.utynote.components.search.model.geojson.entities;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Lang {

    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("iso6391")
    @Expose
    public String iso6391;
    @SerializedName("iso6393")
    @Expose
    public String iso6393;
    @SerializedName("defaulted")
    @Expose
    public Boolean defaulted;

}

File: app/src/main/java/com/utynote/components/search/data/geojson/entities/Lang.kt
package com.utynote.components.search.data.geojson.entities

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Lang {

    @SerializedName("name")
    @Expose
    var name: String? = null
    @SerializedName("iso6391")
    @Expose
    var iso6391: String? = null
    @SerializedName("iso6393")
    @Expose
    var iso6393: String? = null
    @SerializedName("defaulted")
    @Expose
    var defaulted: Boolean? = null

}
